src/components/Splash.js

```diff
@@ -132,6 +133,20 @@ margin-bottom: 1rem;
 }
 `
 
+const GuildCardRootStyle = {
+  minWidth: `256px`,
+}
```

Comment:
This should be another `styled-component`:

```suggestion
const GuildCardRoot = styled.div`
  min-width: 256px;
`
```

Reply:
Fixed in 1359ff2